docs/algorithms/post_layout_optimization.rst

```diff
@@ -7,18 +7,22 @@ Optimizing 2DDWave-clocked Cartesian Layouts
 
 This algorithm aims to decrease the overall layout area of a given 2DDWave-clocked Cartesian layout that has been
 generated using either heuristic methods or machine learning techniques. It achieves this objective by strategically
-repositioning gates within the layout, removing excess wiring, and effectively relocating outputs to more favorable
-positions.
+repositioning gates within the layout, removing excess wiring using the ``wiring_reduction`` algorithm (cf. :ref:`wiring_reduction`), and effectively
+relocating outputs to more favorable positions.
 
 .. tabs::
     .. tab:: C++
         **Header:** ``fiction/algorithms/physical_design/post_layout_optimization.hpp``
 
+        .. doxygenstruct:: fiction::post_layout_optimization_params
+           :members:
         .. doxygenstruct:: fiction::post_layout_optimization_stats
            :members:
-        .. doxygenfunction:: fiction::post_layout_optimization
+        .. doxygenfunction:: fiction::post_layout_optimization(const Lyt& lyt, post_layout_optimization_params ps = {}, post_layout_optimization_stats* pst = nullptr)
```

Comment:
As far as I know, the specialization here is only necessary if there are multiple implementations with the same function name